edsc/collections-collapsed/src/js/edsc-collections-collapsed.coffee

```diff
@@ -44,7 +44,7 @@ class CollectionsCollapsedModel extends KnockoutComponentModel
 
   toggleFlyout: (context, e) =>
     target = @_getFlyoutTarget(e)
-    if domData.get(target, 'flyout') || dom.hasClass(target, 'flyout-visible')
+    if dom.hasClass(target, 'flyout-visible')
```

Comment:
That's what I was saying above about the memory leak. If you close it the way you did above, the domData remains and keeps listening to events because it's never destroyed.